From 70f5a83e5b72562cee565c126a51e7829070880f Mon Sep 17 00:00:00 2001
From: Andrey Okoshkin <a.okoshkin@omprussia.ru>
Date: Fri, 2 Oct 2020 00:25:03 +0300
Subject: [PATCH] [seccomp] Remove statx syscall from tracker-seccomp

statx syscall was added in kernel 4.11, library support was added in glibc 2.28.

Signed-off-by: Andrey Okoshkin <a.okoshkin@omprussia.ru>
---
 src/libtracker-miners-common/tracker-seccomp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libtracker-miners-common/tracker-seccomp.c b/src/libtracker-miners-common/tracker-seccomp.c
index dbaf5c991..88af44370 100644
--- a/src/libtracker-miners-common/tracker-seccomp.c
+++ b/src/libtracker-miners-common/tracker-seccomp.c
@@ -97,7 +97,6 @@ tracker_seccomp_init (void)
 	ALLOW_RULE (statfs64);
 	ALLOW_RULE (lstat);
 	ALLOW_RULE (lstat64);
-	ALLOW_RULE (statx);
 	ALLOW_RULE (access);
 	ALLOW_RULE (getdents);
 	ALLOW_RULE (getdents64);
-- 
2.28.0

